crates/nu-cli/src/app.rs

```diff
@@ -136,6 +195,21 @@ impl App {
             return crate::run_script_file(context, opts);
         }
 
+        if self.perf() {
+            // start a new spit
+            eprintln!(
+                "run_script_file commands 1: {:?}",
```

Comment:
maybe `run commands (-c):`

Reply:
ya, i just discovered this was for running the commands with -c. Is this too verbose `commands passed with -c at launch:`